### ISSUE TITLE
Using static assets?

### ISSUE THREAD

**Issue by kykungz** [doczjs/docz#1104]
## Question

How can I make `docz` to work with components which are using static assets?

I'm trying to use `docz` with `NextJS`. I have these follow folders:
```
/root
├── components
│   └── Image.jsx
└── static
    └── images
        └── car.png
```

```jsx
export default () => (
  <div>
    <img src="/static/images/car.png" />
  </div>
)
```

**Comment by rakannimer** [comment]
Hey @kykungz 

Could you try setting [`public`](https://www.docz.site/docs/project-configuration#public) in your `doczrc.js` to `static` ?

**Comment by kykungz** [comment]
@rakannimer  tried setting `public` to `/static`, `static`, both not working

```js
// doczrc.js
export default {
  public: 'static',
}
```

```js
// doczrc.js
export default {
  public: '/static',
}
```

**Comment by rakannimer** [comment]
Hey,

Thanks for reporting this issue. 
This was fixed starting from [version 2.0.0-rc.44](https://github.com/doczjs/docz/releases/tag/v2.0.0-rc.44).

You can check an example of using images in docz from mdx and jsx files here : https://github.com/doczjs/docz/tree/master/examples/images
